cms/djangoapps/contentstore/views/course.py

```diff
@@ -909,6 +909,8 @@ def settings_handler(request, course_key_string):
                 settings.FEATURES.get('ENABLE_MKTG_SITE', False)
             )
 
+            enrollment_editable = GlobalStaff().has_user(request.user) or not settings.FEATURES.get('IS_EDX_DOMAIN')
```

Comment:
Is IS_EDX_DOMAIN False for Edge? What about microsites? Based on my previous knowledge of the code, I was expecting the check to be a combination of request.user.is_staff and about_page_editable.